source/in_the_box.rst

```diff
@@ -107,20 +108,25 @@ and network is setup for multicast DNS (mDNS) then you may be able to use
 ``fetchXYZ.local`` as the network name where XYZ will be the serial number
 of your robot (remove any leading zeros from the serial number).
 
-Then create your user account if you have not already done so as shown
-below.
-
-.. include:: computer.rst
-   :start-after: embed-user-accounts-start
-   :end-before: embed-user-accounts-end
-
-Connecting a Monitor
+Connecting a Monitor (Optional)
 ~~~~~~~~~~~~~~~~~~~
-If you are unable to access the robot through ssh due to your network settings you will need to connect an HDMI monitor, USB keyboard and USB mouse to the side panel ports. Then use them to select your network as you would a normal Ubuntu computer. Once it is set it should remain connected through reboots.
+If you are unable to access the robot through ssh due to your network settings you will need to connect an HDMI monitor, USB keyboard and USB mouse to the side panel ports. Then use them to select your network from the networking menu, it looks like an empty quarter circle pointed downward when disconnected, in the upper right hand corner of the screen. Once it is set it should remain connected through all reboots.
```

Comment:
while editing this, we should wrap this line at 80 chars for readability of future changes